gradle/libs.versions.toml

```diff
@@ -66,8 +66,7 @@ lifecycle-runtime-ktx = "androidx.lifecycle:lifecycle-runtime-ktx:2.7.0"
 lifecycle-viewmodel-compose = "androidx.lifecycle:lifecycle-viewmodel-compose:2.7.0"
 material = "com.google.android.material:material:1.11.0"
 navigation-compose = "androidx.navigation:navigation-compose:2.7.7"
-openfeedback = "io.openfeedback:openfeedback:0.2.1"
-openfeedback-m3 = "io.openfeedback:openfeedback-m3:0.2.1"
+openfeedback-m3 = "io.openfeedback:openfeedback-m3:0.2.2"
```

Comment:
Fixes a bug when opening a session that had no votes at all, the UI would keep waiting forever